site/src/constants.js

```diff
@@ -19,6 +19,10 @@ export const pinata = {
   jwt: PINATA_JWT,
 }
 
+export const ipfs = {
+  host: IPFS_HOST,
+}
```

Comment:
@Gozala @alanshaw want to raise a point here, that for launch we should use a *slightly* better configured go-ipfs node ( with a behind-DNS IP etc )
You *could* aunch with what we have right now, but flagging that a new one is coming EOW/weekend when I get a chance to set it all up.

cc @mikeal